subprojects/testing-jvm/src/integTest/resources/org/gradle/testing/junit/JUnitJdkNavigationTest/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply plugin: 'java'
+
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    testCompile 'junit:junit:4.12'
+    testCompile 'org.apache.fexli:org.osgi.foundation:1.2.0'
```

Comment:
You need to run `./gradlew integTest` to run integration Tests. Since running all tests will take a very long time I suggest prefixing the task with the sub project: `./gradlew testingJvm:integTest`.